docs/source/tools/trigger-service.rst

```diff
@@ -0,0 +1,123 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Trigger Service
+###############
+
+**WARNING:** The triggers-service is an early access feature that is actively being designed and is subject to breaking changes. The documentation here at this time is limited to basic usage. As more features become available the documentation will be updated to include them. We welcome feedback about the trigger-service on our issue tracker, our forum, or on Slack.
+
+The `DAML triggers <../triggers/index.html#running-a-daml-trigger>`_ documentation shows a simple method using the ``daml trigger`` command to arrange for the execution of a single trigger. Using this method, a dedicated process is launched to host the trigger.
+
+Complex workflows can require running many triggers for many parties and at a certain point, use of ``daml trigger`` with its process per trigger model becomes unwieldy. The trigger-service provides the means to host multiple triggers for multiple parties running against a common ledger in a single process and provides a convenient interface for starting, stopping and monitoring them.
```

Comment:
```suggestion
Complex workflows can require running many triggers for many parties and at a certain point, use of ``daml trigger`` with its process per trigger model becomes unwieldy. The Trigger Service provides the means to host multiple triggers for multiple parties running against a common ledger in a single process and provides a convenient interface for starting, stopping and monitoring them.
```
or different capitalization but this should be consistent with the title imho. Same thing fr a couple of references below.

Reply:
Done holistically.